### ISSUE TITLE
Can't make it work with latest Greenlock

### ISSUE THREAD

**Issue by bushev** [cderche/greenlock-storage-s3#9]
Hello!

I'm trying to use this module to provide S3 storage for accounts & certificates:

```javascript
require('greenlock-express').init(() => {

    greenlock = require('@root/greenlock').create({
      maintainerEmail: MAINTAINER_EMAIL,
      packageRoot: __dirname
    });

    greenlock.manager.defaults({
      subscriberEmail: MAINTAINER_EMAIL,
      agreeToTerms: true,
      challenges: {
        'http-01': {
          module: 'acme-http-01-standalone',
        },
      },
      store: {
        module: 'greenlock-storage-s3',
        accessKeyId: process.env.PSP_APP_AWS_ACCESS_KEY_ID,
        secretAccessKey: process.env.PSP_APP_AWS_SECRET_ACCESS_KEY,
        bucketRegion: process.env.PSP_APP_AWS_DEFAULT_REGION,
        bucketName: `project8-letsencrypt-${process.env.NODE_ENV}`,
        configDir: 'acme/',
        accountsDir: 'accounts/',
        debug: true,
      },
    });

    return {
      greenlock,
      notify: (level, data) => {
        console.log(level, data);
      },
      debug: process.env.NODE_ENV === 'dev',
      cluster: false,
    };
  },
).serve(app);
```

`check.js` file looks incomplete, is this a beta (unfinished) module for S3?

```javascript
module.exports.check = (opts, options) => {
    console.log("accounts.check", opts.account.id);
};
```

![image](https://user-images.githubusercontent.com/7535766/73227623-11c52080-4142-11ea-9ed8-89de754a9322.png)

**Comment by bushev** [comment]
Fixed by #7

**Comment by shyjal** [comment]
Please merge #7  to master and npm. The issue is being faced by many of us @cderche
